jazzmin/templatetags/jazzmin.py

```diff
@@ -417,3 +418,8 @@ def style_bold_first_word(message: str) -> str:
     message = " ".join([word for word in message_words])
 
     return format_html(message)
+
+
+@register.filter
+def unicode_slugify(message: str) -> str:
+    return defaultfilters.slugify(unidecode.unidecode(message))
```

Comment:
hi @fengyouchao ! First of all, thanks for the hard work finding the issue and getting this to work! 🥳 


I think we could simplify that a bit and use: 

`slugify(message, allow_unicode=True)`

That way we wouldn't have to use unidecode I think. I'll perform the changes, but could you double check just in case I'm not reproducing the bug in the same way you are?

Thanks a lot for that!